test/page-test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+
+import { Page, Button } from '../src';
+
+describe('<Page/>', function() {
+  it('Page has title', function() {
+    const component = shallow(<Page title='text'/>);
+    expect(component.find('h1').hasClass('al-title')).to.equal(true);
+  });
+
+  it('Page does not have title', function() {
+    const component = shallow(<Page/>);
+    expect(component.find('h1')).to.have.length(0);
+  });
+
+  it('Title text is being rendered', function() {
+    const component = shallow(<Page title='Testing text'/>);
+    expect(component.find('h1').text()).to.equal('Testing text');
+  });
+
+  it('Action bar renders text', function() {
+    const component = shallow(<Page actionBar='Testing text'/>);
+    expect(component.containsMatchingElement('Testing text')).to.equal(true);
```

Comment:
I believe a string is considered a React Element - just like you can render a string as a child element the same as another element, for example a `<td>` could have a string as a child or a `<Button />`.

I think this could be rewritten as (untested, but looks right from the docs) `expect(component.find('Col[align=right]').text()).to.contain('Testing text');`

I think technically your test would work, but the text would include all text in the component (including any children), not just in the action bar.  In that test case, we aren't passing any children so there wouldn't be any additional text, but I think it is better to be more specific in the test.